src/test/java/tests/administrator/AdministratorPageTest.java

```diff
@@ -0,0 +1,94 @@
+package tests.administrator;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.SignInPage;
+import pages.administrator.AdministratorPage;
+import pages.administrator.order.OrderComponent;
+import tests.BaseTest;
+import utility.DataBaseConnection;
+
+import java.time.Duration;
+
+public class AdministratorPageTest extends BaseTest {
+
+    private final String administratorEmail = "eringonzales@test.com";
+    private final String administratorPassword = "1";
+    private final String orderId = "1";
+    private final String ADMINISTRATOR_URL = "http://miniserver:8880/administrator-panel";
+    private final String SQL_FILE = "src/test/resources/datasource/administrator/administrator_datasource.sql";
```

Comment:
May be we can move this Strings to properties file?

Reply:
It is not a config for the application,  it only used in this class